forum/communications/consumers.py

```diff
@@ -0,0 +1,52 @@
+from asgiref.sync import async_to_sync
+from channels.generic.websocket import JsonWebsocketConsumer
+
+from users.models import CustomUser
+
+
+class ChatConsumer(JsonWebsocketConsumer):
+    def connect(self):
+        self.room_name = self.scope["url_route"]["kwargs"]["room_name"]
+        self.room_group_name = f"chat_{self.room_name}"
+
+        print(self.scope["user"])
+
+        print(self.channel_name)
```

Comment:
Using `print` statements for debugging is not recommended for production code.  Use logging instead.